unison-src/transcripts-using-base/serialized-cases/case-04.v4.hash

```diff
@@ -1 +1 @@
-EXAQLMU6IKGAY7DNOHND5VUQQAQPIJN3IVCF5DISOOEVLRQZ3Q2CZOYEVDMY7MYQX2CG6CJFH2HQD6XOMKHQNK5JUZB3G7RZQNREQRQ=
\ No newline at end of file
+KPXQRV7VO4LIVLHVTR5HXUDOXSFFF2C4QM7IMLCU3YZXJSTJSFDT7YUZCWEW4TLH4SHAHT7NHB2ZTUUWGRGUTWGBVY7BFA33XWRBDKQ=
```

Comment:
Why do you think this changed?

Reply:
Huh, I have no idea how those got regenerated... I reverted both

Reply:
It was two files that I regenerated somehow, the input and the output

Reply:
What's still happening?